openapi_python_client/templates/model.pyi

```diff
@@ -38,13 +38,18 @@ class {{ model.reference.class_name }}:
     {% endif %}
 
 
-    def to_dict(self) -> Dict[str, Any]:
+    def to_dict(self, _serialize_options: Dict[str, Any]={}) -> Dict[str, Any]:
         {% for property in model.required_properties + model.optional_properties %}
         {% if property.template %}
-        {% from "property_templates/" + property.template import transform %}
-        {{ transform(property, "self." + property.python_name, property.python_name) | indent(8) }}
+        {% import "property_templates/" + property.template as prop_template %}
+        {% if prop_template.transform_with_options %}
+        {{ prop_template.transform_with_options(property, "self._" + property.python_name, property.python_name,
+                                                {"serialize_options_expr": "_serialize_options"}) | indent(8) }}
```

Comment:
Here it's passing the _name_ of the new `_serialize_options` parameter, rather than its value, because this is an expression that'll be interpolated into generated code (code that's inside of the `to_dict` method, so it can reference that parameter).